src/QMCDrivers/WFOpt/QMCCostFunctionBase.cpp

```diff
@@ -251,7 +251,7 @@ void QMCCostFunctionBase::reportParametersH5()
   if (!myComm->rank())
   {
     int ci_size = 0;
-    std::vector<opt_variables_type::value_type> CIcoeff;
+    std::vector<opt_variables_type::real_type> CIcoeff;
```

Comment:
This is part is not so right in math but that is the current state of the code. Optimized CI coefficients should remain in complex. So optimizing complex CI coefficients needs fix following the complexification of RotatedSPO. Could you open an issue tracking this.

Reply:
done

Reply:
https://github.com/QMCPACK/qmcpack/issues/4820